config/crd/bases/crunchybridge.crunchydata.com_bridgeclusters.yaml

```diff
@@ -167,7 +171,8 @@ spec:
                 type: object
               last_update:
                 description: 'Updated is the last status update from the controller,
-                  does not correlate to cluster.updated_at'
```

Comment:
Yeah, I'd say revert this file to head prior to commit and I'll fix the code comments so that the build process doesn't try to re-add things that don't need to be there.

Reply:
reverted